projects/igniteui-angular/src/lib/core/styles/components/card/_card-theme.scss

```diff
@@ -228,11 +264,17 @@
     }
 
     %igx-card-header-title {
-        margin: $card-heading-margin;
+        @if map-get($theme, variant) != 'bootstrap' {
```

Comment:
Use `==` to make this easier to read and grasp what the default value is. As the else right now, which is bootstrap, reads as the default at a quick glance.